packages/core-app-api/src/app/AppManager.tsx

```diff
@@ -484,6 +492,21 @@ export class AppManager implements BackstageApp {
     return this.apiHolder;
   }
 
+  private getIdentitylessApiHolder() {
+    const registrySansIdentity = new ApiFactoryRegistry();
+
+    // Assume all APIs in the current registry are valid.
+    this.apiFactoryRegistry.getAllApis().forEach(apiRef => {
+      const factory = this.apiFactoryRegistry.get(apiRef);
+      // Add all API factories except identity!
+      if (factory && apiRef !== identityApiRef) {
+        registrySansIdentity.register('default', factory);
```

Comment:
Hm is it ok to make these `default`? The ones that used to be `static` etc are now being demoted, right? Maybe that's ok since the registration phase is over... @Rugvip